ketcher-autotests/tests/Structure-Creating-&-Editing/SMARTS-attributes/Reaction-flags-attributes/reaction-flags-attributes.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { Page, test } from '@playwright/test';
+import {
+  BondTypeName,
+  clickInTheMiddleOfTheScreen,
+  doubleClickOnAtom,
+  pressButton,
+  selectBond,
+  takeEditorScreenshot,
+  waitForAtomPropsModal,
+  waitForPageInit,
+} from '@utils';
+import {
+  checkSmartsValue,
+  checkSmartsWarnings,
+  setReactionFlagExactChange,
+  setReactionFlagInversion,
+} from '../utils';
+
+const defaultFileFormat = 'MDL Molfile V2000';
+const expectedSmarts = '[#6](-[#6])(-[#6])-[#6]';
+
+async function drawStructure(page: Page, numberOfClicks: number) {
+  await selectBond(BondTypeName.Single, page);
```

Comment:
I checked there are already several `drawStructure` function in different test files, maybe we could just keep one in some util or helper class and remove the others?

Reply:
There are different functions for different tasks. I don’t know if we should make universal one and whether it makes sense?)

Reply:
I checked there are like 4 of this same functions while only 2 kinds of this functions with different params are needed. Maybe we can just keep 2 different versions and remove others?